File: src/VueSPA/src/models/SlothModel.ts
import ReactionOnAccidentType from "./ReactionOnAccidentType";
import ServantsType from "./ServantsType";

interface SlothModel {
	passiveRest: boolean,
	reactionOnAccident: ReactionOnAccidentType,
	servants: ServantsType,
	lazyDescription: string
}

export default SlothModel;

File: src/StoringAPI/Models/SlothModel.cs
using StoringAPI.Models.Enums;

namespace StoringAPI.Models
{
    public class SlothModel
    {
        public bool PassiveRest { get; set; }
        public ReactionOnAccidentType ReactionOnAccident { get; set; }
        public ServantsType Servants { get; set; }
        public string LazyDescription { get; set; }
    }
}
